ios/testmanagerd/testlistener.go

```diff
@@ -55,11 +59,20 @@ type TestError struct {
 	Line    uint64
 }
 
-func NewTestListener(logWriter io.Writer, debugLogWriter io.Writer) *TestListener {
+type TestAttachment struct {
```

Comment:
the UTI would also be nice to have to get some context of what is handled here

Reply:
Added